### ISSUE TITLE
Certificat is invalid on iOS 13

### ISSUE THREAD

**Issue by puppeteer701vungle** [alibaba/anyproxy#512]
Certificat is not verified correctly / valid to catch https requests.

#### Which platform are you running AnyProxy
Server running on MacOS and trying to connect through proxy with iPhone iOS 13

#### The version of the AnyProxy
4.1.0, node v8

#### Your expected behavior of AnyProxy
Able to read HTTPS requests

#### The actual behavior of AnyProxy
Not being to able to read HTTPS requests

#### The log of the error
iOS log:
```
Task <EBD811CA-AA6F-42B9-8905-5E9C567D5B3D>.<8> finished with error [-1202] Error Domain=NSURLErrorDomain Code=-1202 "The certificate for this server is invalid. You might be connecting to a server that is pretending to be “ads.matjaz.vungle.com” which could put your confidential information at risk." UserInfo={NSLocalizedRecoverySuggestion=Would you like to connect to the server anyway?, _kCFStreamErrorDomainKey=3, NSErrorPeerCertificateChainKey=(
    "<cert(0x7fb412012200) s: add.com i: AnyProxy>"
```

**Comment by weimi** [comment]
IOS 13 no longer trust CAs using RSA keys smaller than 2048 bits, while anyproxy using 1024 bits.  
anyproxy use node-easy-cert to generate root CAs.  
 The dirty way is to fix bits length in node_modules/_node-easy-cert@1.3.1@node-easy-cert/dist/certGenerator.js
```
-  const keys = forge.pki.rsa.generateKeyPair(1024);
+   const keys = forge.pki.rsa.generateKeyPair(2048);

```
ref
[Requirements for trusted certificates in iOS 13 and macOS 10.15](https://support.apple.com/en-us/HT210176)

**Comment by ottomao** [comment]
released in node-easy-cert@1.3.2
please reinstall AnyProxy.
